MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -1403,6 +1451,13 @@ - (void)asyncUsers:(void (^)(NSArray<MXUser *> * _Nonnull))success failure:(null
     });
 }
 
+- (void)asyncUsersWithUserIds:(NSArray<NSString *> *)userIds success:(void (^)(NSArray<MXUser *> *users))success failure:(nullable void (^)(NSError * _Nonnull))failure
+{
+    dispatch_async(dispatchQueue, ^{
+        success([self loadUsersWithUserIds:userIds]);
```

Comment:
I missed it in the previous PR for async things but call the ```success``` block on the main thread.

There is a kind of contract in the sdk where:
- API must be called from the main thread (unfortunately not always true)
- and the callback block are called on the main thread